go.sum

```diff
@@ -221,6 +221,8 @@ github.com/openshift/api v3.9.1-0.20191002145753-bb291c9def6c+incompatible h1:HF
 github.com/openshift/api v3.9.1-0.20191002145753-bb291c9def6c+incompatible/go.mod h1:dh9o4Fs58gpFXGSYfnVxGR9PnV53I8TW84pQaJDdGiY=
 github.com/openshift/api v3.9.1-0.20191028134408-abf28f93703b+incompatible h1:1mW1j1mik2++Z+ehEaqm2xcWGiGYJUK8SdbkTgTS+N4=
 github.com/openshift/api v3.9.1-0.20191028134408-abf28f93703b+incompatible/go.mod h1:dh9o4Fs58gpFXGSYfnVxGR9PnV53I8TW84pQaJDdGiY=
+github.com/openshift/api v3.9.1-0.20191105205100-610e47be70db+incompatible h1:HVPbu7fa4U8JUNuLNkXOXkhl/YvVk9xgj1r0DRy902Y=
+github.com/openshift/api v3.9.1-0.20191105205100-610e47be70db+incompatible/go.mod h1:dh9o4Fs58gpFXGSYfnVxGR9PnV53I8TW84pQaJDdGiY=
```

Comment:
@Miciah  do you think the `
[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for CRD with validation schema [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s] ` failure is associated to the api bump?

Why are two `github.com/openshift/api` required here?

Reply:
> @Miciah do you think the `[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for CRD with validation schema [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]` failure is associated to the api bump?

No, I see several reports for the same failure:

https://bugzilla.redhat.com/show_bug.cgi?id=1760198
https://bugzilla.redhat.com/show_bug.cgi?id=1754091
https://bugzilla.redhat.com/show_bug.cgi?id=1761043

Furthermore, https://search.svc.ci.openshift.org/?search=works%20for%20CRD%20with%20validation%20schema shows 161 matches.

> Why are two `github.com/openshift/api` required here?

`go mod` added them.  I'll see whether I can remove them...

Reply:
I ran `go mod tidy`, which cleaned out a lot of old entries from `go.mod`.

Reply:
As for why there are two lines for the same package, https://golang.org/cmd/go/#hdr-Module_authentication_using_go_sum explains:

> The first line gives the hash of the module version's file tree. The second line appends "/go.mod" to the version and gives the hash of only the module version's (possibly synthesized) go.mod file.